### ISSUE TITLE
Cannot install on Windows 10

### ISSUE THREAD

**Issue by TheGreatRambler** [adobe-research/theseus#90]
OS: Windows 10
Version: Version on brackets registry
Brackets version: 1.13

Problem: Whenever I attempt to install theseus-brackets, i get the error: "Extension failed on startup" or something like that. I notice in the devtools, it says, 
```
C:/Users/[MY USERNAME]/AppData/Roaming/Brackets/extensions/user/theseus - Error: Module name "view/PanelManager" has not been loaded yet for context: _
```
I would really like this issue fixed because this extension looks like it hols lots of promise.

**Comment by TheGreatRambler** [comment]
It seems only one change needs to be made. "view/PanelManager" in "FileCallGraphPanel.js" needs to be replaced with "view/WorkspaceManager".

**Comment by TheGreatRambler** [comment]
Yes, i can prove at this time that that is the only change that needs to be made, so if that could be put into the main branch, that would be good.

**Comment by milosvesovic** [comment]
> 
> 
> It seems only one change needs to be made. "view/PanelManager" in "FileCallGraphPanel.js" needs to be replaced with "view/WorkspaceManager".

hey can you tell me where is view/PanelManager. I looked in brackets_extension_folder/theseus but can't find it

**Comment by izabala** [comment]
Download the file: Theseus-master.zip then uncompressed it and then open the file (with a plain text reader or a javascript IDE): theseus-master/src/FileCallGraphPanel.js  and change the line that has "view/PanelManager" and change it change to "view/WorkspaceManager"... in my case it was:
 var PanelManager    = brackets.getModule("view/WorkspaceManager");
Save the file... and compress it againg (so you should have a nuew theseus-master.zip file). 
Open brackets, the extension Manager... and drag and drop the new theseus-master.zip to the botton-left corner of that windows.. (it says Drag .zip here or install from URL...).  It should install without problem!

**Comment by AdvancedNewbie** [comment]
Thanks, this worked for me as well.
